app/javascript/onboarding/components/ClosingSlide.jsx

```diff
@@ -119,35 +123,35 @@ class ClosingSlide extends Component {
       return (
         <div className="onboarding-what-next">
           <a href="/welcome" data-no-instant>
-            Join the Welcome Thread
-            <p className="whatnext-emoji">
-              <span role="img" aria-label="tada">
+            <p>
+              <span className="whatnext-emoji" role="img" aria-label="tada">
```

Comment:
@vaidehijoshi Sounds good.

Since we will shortly deprecate this page anyway and move it to the **[onboarding task card on feed](https://github.com/thepracticaldev/dev.to/issues/6552)**, can we leave this here for now and handle this in that other workflow? The `aria-label` here was at it was when I came to edit this file.

Reply:
yes! I think that makes sense, definitely don't need to block for address this since we're moving away from it, good point. do you want to add that to the appropriate issue in the onboarding project board?

Reply:
@vaidehijoshi will do! I'll add this note and then close this comment area.